src/agent/src/rpc.rs

```diff
@@ -124,6 +125,19 @@ pub fn verify_cid(id: &str) -> Result<()> {
     }
 }
 
+// Partially merge an OCI process specification into another one.
+fn merge_oci_process(target: &mut oci::Process, source: &oci::Process) {
+    if target.args.is_empty() && !source.args.is_empty() {
+        target.args.append(&mut source.args.clone());
+    }
+
+    if target.cwd.is_empty() && !source.cwd.is_empty() {
+        target.cwd = String::from(&source.cwd);
+    }
+
+    target.env.append(&mut source.env.clone());
```

Comment:
OOI, why no check here as is done for `args` and `cwd`?

Reply:
`args` and `cwd` are the command to be executed by the container, there can be only one, and it should be mainly input by the user, and only use the one in Image if the user didn't input it.